docs/testplan.md

```diff
@@ -490,6 +510,26 @@ With the previous role you created from `Strategy Reason`, change `request_acces
 - [ ] Verify that account is locked after several unsuccessful attempts
 - [ ] Verify that redirect to original URL works after successful login
 
+
+## Multi-factor Authentication (mfa)
+Create/modify `teleport.yaml` and set the following authentication settings under `auth_service`
+```
+authentication:
+  type: local
+  second_factor: optional
```

Comment:
You also have to configure the `u2f` section for this: https://goteleport.com/docs/access-controls/guides/u2f/#enable-u2f-support

Reply:
added